content/guides/core-concepts/introduction-to-cypress.md

```diff
@@ -791,43 +815,54 @@ commands that were enqueued using the `cy.*` command chains.
 
 #### Let's take another look at an example
 
+:::visit-mount-test-example
+
 ```js
-it('changes the URL when "awesome" is clicked', () => {
-  cy.visit('/my/resource/path') // 1.
+cy.visit('/my/resource/path') // 1.
+```
 
-  cy.get('.awesome-selector') // 2.
-    .click() // 3.
+```js
+cy.mount(<MyComponent />) // 1.
```

Comment:
Can you either remove the `.` from after the `1` or add `.` after numbers 2-5 below?